app/letters/utils.py

```diff
@@ -79,4 +80,8 @@ def get_letter_pdf(notification):
 
 
 def is_precompiled_letter(template):
-    return template.hidden and template.name == current_app.config['PRECOMPILED_TEMPLATE_NAME']
```

Comment:
I think this belongs in the data model, if it's a property it should be included in the marshmallow schema without a change.

Reply:
I guess it could go in to the data model for the Template class just as a getter.
Will refactor and push through after Richards bug fix is deployed